library/Backends/llvm/test/Test/Codegen.hs

```diff
@@ -17,4 +41,12 @@ tests =
   [trivialLLVMCodegenTest]
 
 trivialLLVMCodegenTest :: TestTree
-trivialLLVMCodegenTest = testCase "Trivial LLVM codegen test" $ True @?= True
+trivialLLVMCodegenTest = testCase "Trivial LLVM codegen test" $ do
+  let testVal = 42
+      ty = ErasedAnn.PrimTy $ Primitive.PrimTy $ AST.IntegerType {typeBits = 8}
```

Comment:
Don't we have `int8` in one of our aliases or `integer` in our compiled code?

Reply:
Oh yes, we have `i8` and `i16`.  I also added a test interface that takes `typeBits` instead -- and, now that I read your comment, I realize that I forgot to use it in a bunch of places!  I added https://github.com/anoma/juvix/pull/1106/commits/2203b0592f7b76d3d4a5238bc79f7d3f997dd1b5 to shorten the test code a lot by using the convenience interfaces.

I also noticed while going over the test code for commonality to factor out that I'd left in the printing of the generated LLVM module, which was useful for debugging but not during automated test execution.  I removed that in https://github.com/anoma/juvix/pull/1106/commits/9e5064fa069109a15ff8a1d56ad2b9d3cf6ac489.